mitodl/micromasters
# Fixed bug involving exam no-shows

#### What are the relevant tickets?
Fixes #3084 

#### What's this PR do?
Fixes bug involving exam no-shows

#### How should this be manually tested?
Using the example no-show pearson file, the following code block should yield a result with no error messages

```python
FILE = 'noshow.dat'
from exams.pearson.readers import EXAMReader
from exams.pearson.download import ArchivedResponseProcessor
from exams.pearson import sftp
process_result = None
with sftp.get_connection() as sftp_connection, open(FILE, 'r') as f:
    processor = ArchivedResponseProcessor(sftp_connection)
    process_result = processor.process_exam_file(f)
process_result
```
